borgmatic/config/schema.yaml

```diff
@@ -945,6 +945,76 @@ properties:
         description: |
             Support for the "borgmatic bootstrap" action, used to extract
             borgmatic configuration files from a backup archive.
+    influxdb_databases:
+        type: array
+        items:
+            type: object
+            required:
+                - token
+            additionalProperties: false
+            properties:
+                token:
+                    type: string
+                    description: |
+                        Token to authenticate request. Required to use InfluxDB.
+                hostname:
+                    type: string
+                    description: |
+                        HTTP address of InfluxDB.
+                    default: http://127.0.0.1:8086
```

Comment:
FYI I don't think these defaults are currently consumed by borgmatic, so they might not make it into the `borgmatic config generate` output. It'd be good to test that.